app/uk/gov/hmrc/preferenceschangednotifier/controllers/model/PreferencesChangedRequest.scala

```diff
@@ -31,9 +32,7 @@ case class PreferencesChangedRequest(
 )
 
 object PreferencesChangedRequest {
-  implicit val mdFormat: Format[MessageDeliveryFormat] =
-    MessageDeliveryFormat.format
-  implicit val dateTimeFormat: Reads[Instant] = RestInstantFormat.reads
-  implicit val reads: Reads[PreferencesChangedRequest] =
-    Json.reads[PreferencesChangedRequest]
+  given Format[MessageDeliveryFormat] = MessageDeliveryFormat.given_Format_MessageDeliveryFormat
```

Comment:
suggestion - I think its better to import the given Format[MessageDeliveryFormat] instead of redefining it.

Reply:
Well spotted, thanks!